extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -258,25 +258,27 @@ export async function runRemoteQuery(
     });
 
     const actionBranch = getActionBranch();
-    const workflowRunId = await runRemoteQueriesApiRequest(credentials, actionBranch, language, repoSelection, owner, repo, base64Pack, dryRun);
+    const apiResponses = await runRemoteQueriesApiRequest(credentials, actionBranch, language, repoSelection, owner, repo, base64Pack, dryRun);
```

Comment:
Minor: there's still 1 API response, it just happens that we return 2 pieces of data from that response. So `apiResponse` would be the correct name IMO.